mysql/toddl.go

```diff
@@ -191,3 +192,35 @@ func cvtPrimaryKeys(conv *internal.Conv, srcTable string, srcKeys []schema.Key)
 	}
 	return spKeys
 }
+
+func cvtForeignKeys(conv *internal.Conv, srcTable string, srcKeys []schema.ForeignKey) []ddl.Foreignkey {
+	var spKeys []ddl.Foreignkey
+	for _, key := range srcKeys {
+		if len(key.Columns) != len(key.ReferColumns) {
+			conv.Unexpected(fmt.Sprintf("ConvertForeignKeys: columns and referColumns don't have the same lengths: len(columns)=%d, len(referColumns)=%d for source table: %s, referenced table: %s", len(key.Columns), len(key.ReferColumns), srcTable, key.ReferTable))
+			continue
+		}
+		spReferTable, err := internal.GetSpannerTable(conv, key.ReferTable)
+		if err != nil {
+			conv.Unexpected(fmt.Sprintf("Can't map foreign key for source table: %s, referenced table: %s", srcTable, key.ReferTable))
+			continue
+		}
+		var spCols, spReferCols []string
+		for i, col := range key.Columns {
+			spCol, err1 := internal.GetSpannerCol(conv, srcTable, col, false)
+			spReferCol, err2 := internal.GetSpannerCol(conv, key.ReferTable, key.ReferColumns[i], false)
+			if err1 != nil || err2 != nil {
+				conv.Unexpected(fmt.Sprintf("Can't map foreign key for table: %s, referenced table: %s, column: %s", srcTable, key.ReferTable, col))
+				continue
+			}
+			spCols = append(spCols, spCol)
+			spReferCols = append(spReferCols, spReferCol)
+		}
+		spKey := ddl.Foreignkey{Columns: spCols,
```

Comment:
Suggest reformatting:

```
		spKey := ddl.Foreignkey{
			Name:         key.Name,
			Columns: spCols,
			ReferTable:   spReferTable,
			ReferColumns: spReferCols}
```

Reply:
formatted as mentioned for mysql as well as postgres